quickcheck-io-sim-compat/test/Spec/DynamicLogic/RegistryModel.hs

```diff
@@ -126,15 +126,15 @@ instance StateModel RegState where
   precondition s (Successful act) = precondition s act
   precondition _ _ = True
 
-  postcondition _ Init _ _ = True
-  postcondition s (WhereIs name) env mtid =
+  postcondition _ Init _ _ = checkBool $ True
+  postcondition s (WhereIs name) env mtid = checkBool $
     (env <$> lookup name (regs s)) == mtid
-  postcondition _s Spawn _ _ = True
-  postcondition s (Register name step) _ res =
+  postcondition _s Spawn _ _ = checkBool $ True
+  postcondition s (Register name step) _ res = checkBool $
```

Comment:
Suggestion: Just fill in an example here? I agree we shouldn't bother with the other branches of the alternative.

Reply:
Sure, can do! :)